src/Humanizer.Tests.Shared/Localisation/en-IN/NumberToWordsTests.cs

```diff
@@ -0,0 +1,62 @@
+﻿using Xunit;
+
+namespace Humanizer.Tests.Localisation.enIN
+{
+    [UseCulture("en-IN")]
+    public class NumberToWordsTests
+    {
+        [Theory]
+        [InlineData(0, "")]
+        [InlineData(1, "one")]
+        [InlineData(10, "ten")]
+        [InlineData(11, "eleven")]
+        [InlineData(20, "twenty")]
+        [InlineData(122, "one hundred and twenty two")]
+        [InlineData(3501, "three thousand five hundred and one")]
+        [InlineData(100, "one hundred")]
+        [InlineData(1000, "one thousand")]
+        [InlineData(1001, "one thousand one")]
+        [InlineData(100000, "one lakh")]
+        [InlineData(1000000, "ten lakh")]
+        [InlineData(10000000, "one crore")]
+        [InlineData(100000000, "ten crore")]
+        [InlineData(1000000000, "one hundred crore")]
```

Comment:
The rules IIRC are:
- Upto 99,999 - identical to US place value system.
- Beyond 1E5, step-size is 1E2 (and not 1E3 like US). 
- 1,00,000 = 1 `lakh`
- 1,00,00,000 = 1 `crore` = 100 `lakh`
- 1,00,00,00,000 = 1 `arab` = 100 `crore`
- 1,00,00,00,00,000 = 1 `kharab` = 100 `arab`
- There are more of these - far less used ones (and less well-known), like `nil` (100 `kharab`, pronounced like *neel*), `padma` (*100 `nil`*) etc. 

The key challenge when building software is deciding how to deal with larger values like `nil`, `padma` etc. They're rarely used, and in practice native speakers simply express `1000 * 1 kharab` as `thousand kharab` (instead of `10 nil`). 

Choosing between pedantic accuracy vs. comprehension is a design-choice software has to make when dealing with numbers > *99 `kharab`*. 

In practice, the usage frequency (as well as familiarity) of `arab`, `kharab` (and larger ones) drops - `crore` is the most commonly used/known one. An argument could be made that usable design would stop at `crore` and ignore `arab` and larger names. That said, `arab`, `kharab` are in extant use (both my experience/memory; I also checked with couple of native Hindi speakers to confirm). IMHO think the right balance would be to retain until `kharab`. 

==
Assuming the choice is to incorporate `arab` and `kharab`, I believe `1000000000` should be `one arab` (rather than `one hundred crore`).

Reply:
Thanks for the analysis @vatsan-madhavan! 

@biju-ps Could you please update and we can get this merged?

Reply:
Thanks for following up on this. What a bummer about COVID-19 - glad to see that you're better now!

I agree that this phenomenon (of expressing everything in terms of `lakh`, and `crore`, and not leveraging `arab`, `kharab`) is not new - this is in line with my experience, and also a couple of folks I spoke about this (to confirm my assumptions, before I posted my earlier comment) thought so as well.

It's not very different than saying that many (but not all) are familiar with `quadrillion` (which comes after `trillion`), but few rarely use it. `quintillion` etc. are far less known, and very rarely used. That said, the question becomes something like this - in a programming library, should one emit `ten thousand trillion`, or should the library emit `10 quadrillion`. This is ultimately a value judgment and depends on your design principles and use-goals.

If you and @clairernovotny decide to represent everything in terms of `crore`, I don't think it would be *wrong* per-se - just a different design choice.

Perhaps what you need here is the ability to switch between *strict* and *common speech* modes, each with different outputs.

Reply:
@vatsan-madhavan thanks for the explanation. I think we can merge this as-is and create a follow-up issue to support the two different modes if someone wants to tackle it.